src/views/organisations/Edit.vue

```diff
@@ -91,10 +95,10 @@ export default {
           if (data.url === this.organisation.url) {
             delete data.url;
           }
-          if (data.email === (this.organisation.email || '-')) {
+          if (data.email === (this.organisation.email || "-")) {
             delete data.email;
           }
-          if (data.phone === (this.organisation.phone || '-')) {
+          if (data.phone === (this.organisation.phone || "-")) {
             delete data.phone;
           }
```

Comment:
Will be an empty string if the input is empty.

```suggestion
          if (data.phone === (this.organisation.phone || "")) {
            delete data.phone;
          }
```

Reply:
Done